x-pack/plugins/observability_solution/observability_ai_assistant_app/server/functions/query/esql_docs/esql-acos.txt

```diff
@@ -2,10 +2,9 @@ ACOS
 
 Syntax
 Parameters
-n
-Numeric expression. If null, the function returns null.
-DescriptionReturns the arccosine of n as an
-angle, expressed in radians.Supported types
+number
+Number between -1 and 1. If null, the function returns null.
+DescriptionReturns the arccosine of n as an angle, expressed in radians.Supported types
```

Comment:
Just noticed that there is no whitespace between the words "Description" and "Returns", do you think that might cause issue? It's in many files.

Reply:
That's OK! The LLM converts these to tokens, and whitespace doesn't matter much.